mysql-test/extra/rpl_tests/rpl_multi_update.test

```diff
@@ -1,6 +1,16 @@
 source include/master-slave.inc;
 
-call mtr.add_suppression('Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT.');
+--connection master
+set session sql_log_bin=0;
+call mtr.add_suppression("Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT");
+set session sql_log_bin=1;
+
+--connection slave
+set session sql_log_bin=0;
+call mtr.add_suppression("Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT");
+set session sql_log_bin=1;
+
```

Comment:
Did you try to use:
```
call mtr.add_suppression('Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT.');
--source include/sync_slave_sql_with_master.inc
--connection master
```

Reply:
I found the reason in your commit message.